test/hotspot/jtreg/compiler/uncommontrap/TestDeoptOOM.java

```diff
@@ -42,7 +42,7 @@
  *      -XX:CompileCommand=exclude,compiler.uncommontrap.TestDeoptOOM::main
  *      -XX:CompileCommand=exclude,compiler.uncommontrap.TestDeoptOOM::m9_1
  *      -XX:+UnlockDiagnosticVMOptions
- *      -XX:+UseZGC -XX:+LogCompilation -XX:+PrintDeoptimizationDetails -XX:+TraceDeoptimization -XX:+Verbose
+ *      -XX:+UseZGC -XX:+LogCompilation -Xlog:disable -Xlog:deoptimization=trace -XX:+TraceDeoptimization -XX:+Verbose
```

Comment:
Why you need `-Xlog:disable` ?

Reply:
Short version: Mistake on my part, it shouldn't be there.

Long version: `-Xlog:disable` disables the default output to stdout, this is important when you want your logs to *not* go to stdout at all. Of course, we want our logs to go to stdout, so this is unnecessary here.